app/Services/Finance/PirepFinanceService.php

```diff
@@ -531,17 +533,21 @@ public function getReconciledFaresForPirep($pirep)
      */
     public function getGroundHandlingCost(Pirep $pirep)
     {
+        // Get Airport GH Costs for Departure and Arrival or revert back to settings
+        $dpt_gh_cost = $pirep->dpt_airport->ground_handling_cost ?? setting('airports.default_ground_handling_cost');
+        $arr_gh_cost = $pirep->arr_airport->ground_handling_cost ?? setting('airports.default_ground_handling_cost');
+        $gh_cost = $dpt_gh_cost + $arr_gh_cost;
```

Comment:
I think it would be better to have two separate charges, for both arrival and departure. Easier to debug in the logs also

Reply:
Then we need two functions or an extended fuction which accepts the airport being checked (dep/arr) ... I can work on it but you may not like the way i code things.